src/juliaexepath.ts

```diff
@@ -80,22 +79,27 @@ export async function getJuliaExePath() {
                     } else {
                         setNewJuliaExePath(p)
                     }
-                    foundJulia = true
                     break
                 }
                 catch (e) {
                 }
             }
-            if (!foundJulia) {
-                setNewJuliaExePath(getExecutablePath())
-            }
```

Comment:
This whole construction was unnecessary: The outer `if` clauses already made sure we can _only_ ever end up here if `getExecutablePath` returns `null` _and_ `actualJuliaExePath` is `null`. In that case we don't need to call `setNewJuliaExePath` at all because it is essentially a no op.